app/assets/stylesheets/sbstyles.scss

```diff
@@ -0,0 +1,26 @@
+body {
+  background-image: image-url("background1.jpg");
+  font-family: "Karma", serif;
+}
+
+h1 {
+  color: #df7401;
```

Comment:
Color literals like `#df7401` should only be used in variable declarations; they should be referred to via variable everywhere else.